package.json

```diff
@@ -407,10 +408,11 @@
     "prepare": "husky install",
     "pretest": "npm run compile && npm run lint",
     "test": "extest get-vscode && extest install-vsix",
+    "ui-test": "extest setup-and-run -i out/client/ui-test/allTestsSuite.js",
     "vscode:prepublish": "npm run webpack",
     "watch": "tsc -b -w",
     "watch:server": "tsc -p ../ansible-language-server --outDir out/server -w",
-    "webpack": "npm run clean && webpack --mode production --config ./webpack.config.js",
+    "webpack": "npm run clean && npm run compile && webpack --mode production --config ./webpack.config.js",
```

Comment:
Why do we need to compile first? Webpack does compilation too.

Reply:
Webpack does not compile ui test related files (*.ts) into *.js. It places only *.ts file under out folder.

Reply:
So webpack-like compilation differs from what ```npm run compile``` does.

Reply:
I'm going to mark this as resolved to unblock the release.